source/wp-content/themes/wporg-parent-2021/sass/blocks/_search.scss

```diff
@@ -16,7 +16,6 @@
 	margin-top: var(--wp--custom--margin--baseline);
 
 	> label {
-		width: 100%;
```

Comment:
Is this in the right PR?

Reply:
Yeah, this causes the content to break out of the screen when the search is in a sidebar.